src/analysis/static/graph/CFG.ts

```diff
@@ -50,4 +51,70 @@ export class CFG {
       .filter((node) => node.type === NodeType.Root)
       .map((node) => <RootNode>node);
   }
+
+  getNodeById(nodeId : string): Node {
+    const found = this._nodes.filter((node: Node) => node.id == nodeId);
+    if(found.length != 1) {
+      console.log("No node with such id in CFG");
```

Comment:
Please refrain from using console.log.
This function should either throw an error when it is not found or simply return null.
in the latter case we do not even need an if-statement.